masterlist.yaml

```
@@ -26760,3 +26760,33 @@ plugins:
         util: 'SSEEdit v4.0.4c'
       - crc: 0x76F44BFE
         util: 'SSEEdit v4.0.4c'
+        
+  - name: 'Solitude Docks.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/13922/'
+        name: 'Solitude Docks'
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/33777/'
+        name: 'Solitude Docks Updated'
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64071/'
+        name: 'Rob''s Bug Fixes - Solitude Docks Updated'
+    msg:
+      - <<: *patchUpdateAvailable
+        subs: [ '[Rob''s Bug Fixes - Solitude Docks Updated](https://www.nexusmods.com/skyrimspecialedition/mods/64071/)' ]
+        condition: '(checksum("Solitude Docks.esp", 650E43BB) or checksum("Solitude Docks.esp", A100B5ED) or checksum("Solitude Docks.esp", DDAD84FE) or checksum("Solitude Docks.esp", 5A146CCB))'
```

The extra parentheses are unnecessary:
```suggestion
        condition: 'checksum("Solitude Docks.esp", 650E43BB) or checksum("Solitude Docks.esp", A100B5ED) or checksum("Solitude Docks.esp", DDAD84FE) or checksum("Solitude Docks.esp", 5A146CCB)'
```